pages/download/index.html

```diff
@@ -23,6 +23,14 @@ <h1>Download a Silverblue image</h1>
 <p>You may then install Silverblue by booting from your USB Drive. <b>Note:</b> it is
 advisable to keep the default partitioning setup that the Installer suggests,
 since rpm-ostree expects this setup.</p>
+<p> You can verify your download with 
+<a href="https://mirrors.huaweicloud.com/repository/fedora/releases/29/Silverblue/x86_64/iso/Fedora-Silverblue-29-1.2-x86_64-CHECKSUM">
```

Comment:
This points to a specific mirror; it should instead point to the redirect URL

https://download.fedoraproject.org/pub/fedora/linux/releases/29/Silverblue/x86_64/iso/Fedora-Silverblue-29-1.2-x86_64-CHECKSUM